src/__tests__/store.providerfactory.test.js

```diff
@@ -1,31 +0,0 @@
-/* eslint-env jest */
-
-describe('#create() Store Provider Factory', () => {
-
-    beforeEach(() => {
-        jest.resetModules();
-
-    })
-
-    it('should return an EventEmitter object for \'aws\'', () => {
-		var EventEmitter = require('events').EventEmitter;
-
-        jest.setMock('../../config', {
-            providerType: 'aws'
-        })
-        var r = require('../store.providerfactory.js').create({})()
```

Comment:
Consider making the first arg to create method a default if it is optional:

```
create (defaultArg = {}) {
...
}
```

Reply:
test needs overhaul